[2025-10-02T06:24:05.904+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T06:24:05.911+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.create_table_if_needed manual__2025-10-02T06:23:19.658047+00:00 [queued]>
[2025-10-02T06:24:05.914+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.create_table_if_needed manual__2025-10-02T06:23:19.658047+00:00 [queued]>
[2025-10-02T06:24:05.914+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T06:24:05.919+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_table_if_needed> on 2025-10-02 06:23:19.658047+00:00
[2025-10-02T06:24:05.923+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=911) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T06:24:05.924+0000] {standard_task_runner.py:63} INFO - Started process 913 to run task
[2025-10-02T06:24:05.924+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'create_table_if_needed', 'manual__2025-10-02T06:23:19.658047+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmpxpka7tj6']
[2025-10-02T06:24:05.925+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask create_table_if_needed
[2025-10-02T06:24:05.945+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.create_table_if_needed manual__2025-10-02T06:23:19.658047+00:00 [running]> on host b4099255ca46
[2025-10-02T06:24:05.976+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='create_table_if_needed' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T06:23:19.658047+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T06:23:19.658047+00:00'
[2025-10-02T06:24:05.977+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T06:24:05.987+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-10-02T06:24:05.987+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-10-02T06:24:05.988+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-10-02T06:24:07.150+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.151+0000] {sql.py:470} INFO - Running statement: USE WAREHOUSE COMPUTE_WH;, parameters: None
[2025-10-02T06:24:07.258+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.259+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T06:24:07.259+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T06:24:07.260+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-908d-0000-00131e678161
[2025-10-02T06:24:07.260+0000] {sql.py:470} INFO - Running statement: USE ROLE ACCOUNTADMIN;, parameters: None
[2025-10-02T06:24:07.356+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.357+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T06:24:07.358+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T06:24:07.358+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-908d-0000-00131e678165
[2025-10-02T06:24:07.358+0000] {sql.py:470} INFO - Running statement: USE DATABASE YOUR_DB;, parameters: None
[2025-10-02T06:24:07.487+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.488+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T06:24:07.489+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T06:24:07.490+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-9174-0000-00131e675165
[2025-10-02T06:24:07.490+0000] {sql.py:470} INFO - Running statement: USE SCHEMA PUBLIC;, parameters: None
[2025-10-02T06:24:07.609+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.611+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T06:24:07.611+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T06:24:07.612+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-8f79-0000-00131e67c115
[2025-10-02T06:24:07.612+0000] {sql.py:470} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS YOUR_DB.PUBLIC.SENTIMENT140_CLEAN (
            LABEL INTEGER,
            TEXT  STRING
        );
        , parameters: None
[2025-10-02T06:24:07.757+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.758+0000] {sql.py:479} INFO - Rows affected: 1
[2025-10-02T06:24:07.759+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-10-02T06:24:07.759+0000] {snowflake.py:408} INFO - Snowflake query id: 01bf6f60-0306-8f88-0000-00131e67d1ad
[2025-10-02T06:24:07.905+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-10-02T06:24:07.906+0000] {connection.py:762} INFO - closed
[2025-10-02T06:24:07.963+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2025-10-02T06:24:08.031+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T06:24:08.034+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T06:24:08.052+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=create_table_if_needed, run_id=manual__2025-10-02T06:23:19.658047+00:00, execution_date=20251002T062319, start_date=20251002T062405, end_date=20251002T062408
[2025-10-02T06:24:08.080+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T06:24:08.089+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
